examples/rfm9x_rpi_interrupt.py

```diff
@@ -0,0 +1,72 @@
+# Simple demo of sending and recieving data with the RFM95 LoRa radio.
+# Author: Tony DiCola
```

Comment:
Please add your name to this too, @jerryneedell.

Reply:
implemented